### DISCUSSION TITLE
Are Saving Session Points the same as the "new" Octoplus Points?

### DISCUSSION THREAD

**Discussion by deltamelter** [BottlecapDave/HomeAssistant-OctopusEnergy#484] [Q&A]
As I joined OE and started using this integration long after the DFS and saving sessions of last year, I have never seen these sensors in use, but now that OE are kicking that off again with Octoplus and new saving sessions including "Surprise Saving Sessions" what will these sensor do? and will you be looking to expand on this for Octoplus?

**Comment by BottlecapDave** [comment]
Hi, I think it's an extension of this. By the looks of it, the saving sessions sensor should still work (I'll try to extend it so you can opt in via HA), but the saving session points endpoint has been retired for something else so will need to fix that.

**Comment by ITSpecialist111** [comment]
MMM, the sensor is saying I have 1 point, however on the app I have around 500 Octopoints. Could be different attribute?

**Reply by BottlecapDave** [reply]
As mentioned above, the octopoints is a different endpoint so will need to be implemented.
